packages/block-library/src/post-excerpt/edit.js

```diff
@@ -146,11 +154,6 @@ export default function PostExcerptEdit( {
 	isSelected,
 	context,
 } ) {
-	if ( ! context.postType || ! context.postId ) {
-		return (
-			<Warning>{ __( 'Post excerpt block: no post found.' ) }</Warning>
-		);
-	}
 	return (
 		<PostExcerptEditor
```

Comment:
It seems that component is useless, we can just remove it and embed everything here?